fmriprep/__init__.py

```diff
@@ -44,5 +44,19 @@ def _new_version():
     iflogger.setLevel(level)
     if v is None:
         iflogger.warn('afni_vcheck executable not found')
-    return v
 Info.version = staticmethod(_new_version)
+
+from .due import due, Url
```

Comment:
Could you move the import to the top? Just above warnings seems reasonable.

And I think we need to pull in the `due.py` stub for this to work. Alternately, drop the `.` and add duecredit to the dependencies.